pichillilorenzo/flutter_inappwebview
# Fix popup webview not loaded

## Connection with issue(s)

Resolve issue #1500 

For some reason, when the web view that is about to be created through the `webView(_ webView: WKWebView,
                        createWebViewWith configuration: WKWebViewConfiguration,
                  for navigationAction: WKNavigationAction,
                  windowFeatures: WKWindowFeatures)` method is a pop up, there is a chance that web view not being loaded (first-time loading) which causes it showing a white blank screen. If we reopen the popup view for second-time, the popup web view will be loaded as usual. 

This PR basically tends to ensure every pop up webview will be loaded ([reference](https://stackoverflow.com/a/27391215))





## Testing and Review Notes
- Comment the changes to original code.
- Run the app with the sample code from #1500 
- Verify that popup cannot be opened after clicking the button. 
- Close the app, uncomment the changes to code.
- Run again and click the button, popup web view should be loaded instead of white blank screen.




## Screenshots or Videos

https://user-images.githubusercontent.com/36991424/215966289-8df762ce-3d93-478a-89bc-0d4688f8f998.mp4





## To Do
